src/gridfs/indexes.ts

```diff
@@ -31,18 +35,24 @@ export async function checkIndexes(
   chunksCollection: Collection<Chunk>,
   hasCheckedIndexes: (value: boolean) => void,
 ) {
-  const filesCollectionIsEmpty = !await filesCollection.findOne({}, {
-    projection: { _id: 1 },
-  });
+  const filesCollectionIsEmpty = !(await filesCollection.findOne(
+    {},
+    {
+      projection: { _id: 1 },
+    },
+  ));
 
-  const chunksCollectionIsEmpty = !await chunksCollection.findOne({}, {
-    projection: { _id: 1 },
-  });
+  const chunksCollectionIsEmpty = !(await chunksCollection.findOne(
+    {},
+    {
+      projection: { _id: 1 },
+    },
+  ));
 
   if (filesCollectionIsEmpty || chunksCollectionIsEmpty) {
     // At least one collection is empty so we create indexes
-    createFileIndex(filesCollection);
-    createChunksIndex(chunksCollection);
+    await createFileIndex(filesCollection);
+    await createChunksIndex(chunksCollection);
```

Comment:
oh yeah and both functions require await but they don't require to be async function

Reply:
I fixed this in the next commit.